ios/FluentUI.Demo/FluentUI.Demo/Demos/NotificationViewDemoController.swift

```diff
@@ -91,54 +91,52 @@ class NotificationViewDemoController: DemoController {
     }
 
     private func createNotificationView(forVariant variant: Variant) -> NotificationView {
-        let view = NotificationView()
         switch variant {
         case .primaryToast:
-            view.setup(style: .primaryToast,
+            return NotificationView.init(style: .primaryToast).setup(style: .primaryToast,
```

Comment:
You can skip the `.init` part for each of these and just initialize with `NotificationView(style: .primaryToast)`.